app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    return sum([player.get_rating() for player in players])
```

Comment:
you can omit here [] brackets to prevent creating a redundant list

Reply:
Thanks!